server/@types/models/Referral.ts

```diff
@@ -2,12 +2,12 @@ import type { CourseOffering } from './CourseOffering'
 import type { Person } from './Person'
 
 type Referral = {
+  additionalInformation: string
```

Comment:
do we want to put this below the `id`? Not a big deal, but if we're disabling member ordering below, maybe feels good to